docs/misc/keymappings.md

```diff
@@ -0,0 +1,150 @@
+  "key.categories.misc"
+)
+```
+
+## Checking a `KeyMapping`
+
+A `KeyMapping` can be checked to see whether it has been clicked. Depending on when, the mapping can be used in a conditional to apply the associated logic.
+
+### Within the Game
+
+Within the game, a mapping should be checked by listening to `ClientTickEvent` on the [**forge event bus**][forgebus] and checking `KeyMapping#consumeClick` within a while loop. `#consumeClick` will return `true` only the number of times the input was performed and not already previously handled, so it won't infinitely stall the game.
+
+```java
+// Event is on the forge event bus only on the physical client
+public void onClientTick(ClientTickEvent event) {
+  if (event.phase == TickEvent.Phase.END) { // Only call code once as the tick event is called twice every tick
+    while (EXAMPLE_MAPPING.get().consumeClick()) {
+      // Execute logic to perform on click here
+    }
+  }
+}
+```
+
+!!! warning
+    Do not use the `InputEvent`s as an alternative to `ClientTickEvent`. There are separate events for keyboard and mouse inputs only, so they wouldn't handle any additional inputs.
+
+### Inside a GUI
+
+Within a GUI, a mapping can be checked within one of the `GuiEventListener` methods using `IForgeKeyMapping#isActiveAndMatches`. The most common methods which can be checked are `#keyPressed` and `#mouseClicked`. 
+
+`#keyPressed` takes in the `GLFW` key token, the platform-specific scan code, and a bitfield of the held down modifiers. A key can be checked against a mapping by creating the input using `InputConstants#getKey`. The modifiers are already checked within the mapping methods itself.
+
+```java
+// In some Screen subclass
+@Override
+public boolean keyPressed(int key, int scancode, int mods) {
+  if (EXAMPLE_MAPPING.get().isActiveAndMatches(InputConstants.getKey(key, scancode))) {
+    // Execute logic to perform on key press here
+    return true;
+  }
+  return super.keyPressed(x, y, button);
+} 
+```
+
+!!! note
+    If you do not own the screen which you are trying to check a key for, you can listen to the `Pre` or `Post` events of `ScreenEvent$KeyPressed` on the [**forge event bus**][forgebus] instead.
+
+`#mouseClicked` takes in the mouse's x position, y position, and the button clicked. A mouse button can be checked against a mapping by creating the input using `InputConstants$Type#getOrCreate` with the `MOUSE` input.
+
+```java
+// In some Screen subclass
+@Override
+public boolean mouseClicked(double x, double y, int button) {
+  if (EXAMPLE_MAPPING.get().isActiveAndMatches(InputConstants.TYPE.MOUSE.getOrCreate(button))) {
+    // Execute logic to perform on mouse click here
+    return true;
+  }
+  return super.mouseClicked(x, y, button);
+} 
+```
+
+!!! note
+    If you do not own the screen which you are trying to check a mouse for, you can listen to the `Pre` or `Post` events of `ScreenEvent$MouseButtonPressed` on the [**forge event bus**][forgebus] instead.
```

Comment:
Given the proximity and similarity of the two notes here, the "mouse" in this note and the "key" in the previous note should be emphasized.